### ISSUE TITLE
Clear multifield performs a reset

### ISSUE THREAD

**Issue by potj69** [tieniber/GridSearch#47]
When I use the X in a multifield to clear the multifield, a total reset is performed. I was expecting that only the multifield I was using the X on is being cleared and the search changed accordingly. Please correct me if I am wrong.

**Comment by NarendranJ3011** [comment]
Yes even I am expecting this issue to occur for me when Click X all the field are getting rest
